core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -586,6 +586,7 @@ class AutoTuner(
     appendRecommendation("spark.rapids.shuffle.multiThreaded.writer.threads", numExecutorCores)
     appendRecommendation("spark.rapids.sql.multiThreadedRead.numThreads",
       Math.max(20, numExecutorCores))
+    appendRecommendation("spark.rapids.sql.batchSizeBytes", BATCH_SIZE_BYTES)
```

Comment:
I really have a problem with this. Just saying that we should set the target batch size to just under 2 GiB with no indication of the amount of memory on the GPU. Not looking at the parallelism setting.  This is just as likely to cause problems as it is to solve them. If we wanted it to be 2 GiB to start with I would have left it when I made the changes

https://github.com/NVIDIA/spark-rapids/pull/8227

What actual benchmarks do we have to show that this is the right setting to make?